addons/stock/report/report_stock_rule.py

```diff
@@ -34,11 +34,11 @@ def _get_report_values(self, docids, data=None):
         for location in locations:
             # TODO: group the RR by location_id to avoid a filtered at each loop
             rr = reordering_rules.filtered(lambda r: r.location_id.id == location.id)
-            if location.putaway_strategy_id or rr:
+            if location.putaway_rule_ids or rr:
                 header_lines[location.id] = {'putaway': [], 'orderpoint': []}
-                if location.putaway_strategy_id:
-                    putaway = location.putaway_strategy_id._get_putaway_rule(product)
-                    header_lines[location.id]['putaway'].append(putaway)
+                if location.putaway_rule_ids:
+                    for putaway in location.putaway_rule_ids:
+                        header_lines[location.id]['putaway'].append(putaway)
```

Comment:
not sure it is acting the same as before

Reply:
I changed that, thanks